### PR TITLE
Fix Alpine package names, add apk command

### PR DESCRIPTION
**Pull request by pevik**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -17,7 +17,7 @@ check:
 
 .PHONY: tags
 tags:
-	ctags -f tags --recurse --totals --extras=+q \
+	ctags -f tags --recurse --totals --extra=+q \
 	    --languages=Perl --langmap=Perl:+.t \
 	    $(perl_src)
```
Comment (metan-ucw) [inline Makefile:23]: Looks like an unrelated change.

File: install_pkg.pm [hunk 2]
```diff
@@ -143,6 +155,10 @@ sub update_pkg_db
 {
 	my ($distro) = @_;
 
+	if ($distro eq "alpine") {
+		return "apk update";
+	}
+
 	if ($distro eq "debian") {
 		return "apt-get update";
 	}
```
Comment (metan-ucw) [inline install_pkg.pm:164]: This part looks good.

File: Makefile (hunk 1 above)
Comment (pevik) [reply Makefile]: Yep, silly error when testing another PR. Removed.

**Comment by pevik** [comment]
It should be ready now.
